src/userEvent/touchEvents.ts

```diff
@@ -0,0 +1,115 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import act from '../act';
+import { getHostParent } from '../helpers/component-tree';
+import { filterNodeByType } from '../helpers/filterNodeByType';
+import { isHostElementPointerEventEnabled } from '../helpers/isHostElementPointerEventEnabled';
+import { getHostComponentNames } from '../helpers/host-component-names';
+import { jestFakeTimersAreEnabled } from '../helpers/timers';
+import { DEFAULT_MIN_PRESS_DURATION } from './constants';
+import { warnAboutRealTimers } from './utils/warnAboutRealTimers';
+
+const defaultPressEvent = {
+  persist: jest.fn(),
```

Comment:
I wonder should we actually provide anything outside of `nativeEvent` key? Can that be useful in both real code and tests. @pierrezimmermannbam wdyt?

Reply:
I have added the minimum so that the implementation worked, persist and currentTarget are both used by React Native in Pressability so that's why they're here. For now there is no option to customize the event. I very rarely needed that option so I'm not sure what the use cases for that would be but it could be available in a v2 if users have a need for it